files/default/neutron-ha-tool.py

```diff
@@ -235,6 +239,7 @@ def run(args):
     qclient.format = 'json'
 
     configure(args, qclient)
+    Configuration.router_filter.load()
```

Comment:
IMO `load` do not need to be exposed in every instance. This can be called in the constructor, or better in the `configure`